account_invoice_report_grouped_by_picking/models/account_move.py

```diff
@@ -48,8 +48,10 @@ def lines_grouped_by_picking(self):
                         qty = so_line.product_uom_qty
                         picking_dict[key] += qty
                         remaining_qty -= qty
+
```

Comment:
Why adding a useless extra empty line. PEP8 discourages this, and I personally hate it.

Reply:
Fixed